Assets/Scripts/DeadEnemy.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DeadEnemy : MonoBehaviour
+{
+    public int health = -1;
```

Comment:
Use serialize field for private fields that need to be exposed to the editor. Use properties for public data other classes need access to.